packages/manager/apps/pci-private-registry/src/components/CIDR/DeleteModal.component.tsx

```diff
@@ -0,0 +1,144 @@
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+import { useCallback, useContext } from 'react';
+import { OsdsModal, OsdsButton } from '@ovhcloud/ods-components/react';
+import {
+  Description,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+import {
+  useIpRestrictionsWithFilter,
+  useUpdateIpRestriction,
+} from '@/api/hooks/useIpRestrictions';
+import {
+  FilterRestrictionsServer,
+  TIPRestrictionsData,
+  TIPRestrictionsDefault,
+  TIPRestrictionsMethodEnum,
+} from '@/types';
+import { categorizeByKey } from '@/helpers';
+import { Context } from '../../pages/CIDR/FilterContext.provider';
+
+type DeleteModalProps =
+  | { all: true; cidr?: never; onClose: () => void }
+  | { all?: false; cidr: TIPRestrictionsData; onClose: () => void };
+
+export default function DeleteModal({
```

Comment:
Maybe, we could reuse `DeleteModal` from `manager-pci-common` library?

Reply:
What about this ? 
There is an existing component from the library, could it make the job?